src/PeopleColumn1.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react'
 import styled from 'styled-components'
-import birthdayData from './birthdays.json'
 import YellowCard from './YellowCard.js'
 import StyledColumn from './StyledColumn'
 
+import firebase from 'firebase/app'
+import 'firebase/database'
+import {
+  FirebaseDatabaseProvider,
+  FirebaseDatabaseNode,
+} from '@react-firebase/database'
+
+import FirebaseConfig from './FirebaseConfig'
+
 const BirthdayList = styled.p`
```

Comment:
suggestion: put the styled component after the export default function

Reply:
Why do you suggest it? Thought it will be nice to have the styling at the end of the component, because it's not that relevant for the functionality.